File: src/javascript/Exponentiation.js
/**
 * @param {number} base 
 * @param {number} exponent 
 */
function exponentiation(base, exponent) {
    const baseNumber = base;
    for (let index = 0; index < exponent-1; index++) 
        base *= baseNumber;

    return base;
}

function main(){
    // Sample: 3³ = 3*3*3 = 27
    console.log('Exponentiation 3³ => ', exponentiation(3, 3));
}

main();

File: src/go/exponentiation.go
package main

import "fmt"

func Exponentiation(base int, exponent int) int {
	for index := 0; index < (exponent - 1); index++ {
		base *= exponent
	}
	return base
}

func main() {
	fmt.Println("Exponentiation:", Exponentiation(5, 5))
}
